README.md

```diff
@@ -6,7 +6,10 @@
 	</p>
 	<p>
 		<a href="https://cuttle.cards" title="Cuttle Website">Website</a> | <a href="https://cuttle-cards.github.io/cuttle/" title="Cuttle Design Guide">Design Guide</a>
-	</p>
+    	</p>
+
+[![Discord Community](https://discord.com/api/guilds/833694928684187669/widget.png?style=banner2)](https://discord.gg/9vrAZ8xGyh) 
```

Comment:
This is pretty good! It can be previewed here, https://github.com/cuttle-cards/cuttle/blob/ef73ddd2c89c11a5d42a1f131c96c6b5547b6289/README.md.

![CFA7346E-6D16-4B9E-9FC2-13E33DF07C5B](https://user-images.githubusercontent.com/414475/194852844-3b73d0bc-96a8-4e40-9320-95ad28e6972f.png)